ipam/allocator.go

```diff
@@ -197,29 +176,46 @@ func hasBeenCancelled(cancelChan <-chan bool) func() bool {
 
 // Actor client API
 
+func (alloc *Allocator) SetDefaultSubnet(subnet address.CIDR) error {
+	resultChan := make(chan error)
+	if subnet.Size() < 4 {
```

Comment:
First and last addresses in a subnet are excluded, so 4 is the smallest size where any allocation will succeed.